src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,15 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sort the list
+     * @return UniquePersonList
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)